src/main/resources/fi/metatavu/famifarm/server/liquibase/changelog.xml

```diff
@@ -239,7 +239,40 @@
       </column>
     </createTable>
   </changeSet>
-      
+  <changeSet id="packing" author="simeon.platonov">
```

Comment:
Do not modify the history in changelog file. It will make it impossible to install update in the production. Add necessary operations to the bottom of the file which will modify the current database to suit new code.